recipe/meta.yaml

```diff
@@ -8,9 +8,15 @@ source:
   fn: PyGithub-{{ version }}.tar.gz
   url: https://github.com/PyGithub/PyGithub/archive/v{{ version }}.tar.gz
   sha256: 862f481d5a51825f1f7abc76d3a17e571f378d6b95c629c581f7eccd3ea04200
+  patches:
+    # Adds a patch that provides the ability to create and modify repo content.
+    # Link to the pull request is shown below.
+    #
+    # ( https://github.com/PyGithub/PyGithub/pull/379 )
+    - PR_379.diff
```

Comment:
I'm relatively uncomfortable with this change - we are essentially railroading the definition of "pygithub" 1.26 to include our own desired functionality.

I completely know the motivation is genuine and not intended to apply undue pressure on the developers, but I can't realistically justify merging the PR. Is there some other way I can help in order to facilitate the underlying motivation?